### DISCUSSION TITLE
Welcome to LightBox_Gallery Discussions!

### DISCUSSION THREAD

**Discussion by aryaparasharmrt** [aryaparasharmrt/LightBox_Gallery#7] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by aryaparasharmrt** [comment]
disscussion added

**Comment by aryaparasharmrt** [comment]
add

**Comment by KarandeepBisht** [comment]
hello

**Comment by aryaparasharmrt** [comment]
Hi

**Comment by aryaparasharmrt** [comment]
This changes was asked by manager. Thankyou

**Comment by KarandeepBisht** [comment]
Hello sir

**Reply by aryaparasharmrt** [reply]
Hello Karan

**Comment by KarandeepBisht** [comment]
please give the Solution for the bug.

**Comment by KarandeepBisht** [comment]
bhai mera new task aa gya
Implement version control for healing practice projects and track progress using GitHub.

**Comment by KarandeepBisht** [comment]
Implement version control for healing practice projects and track progress using GitHub.   

Ye bta kaise krna h ?

**Reply by aryaparasharmrt** [reply]
Repo banani hai ismai isse related and files banake content add kerna hai
